scripts/correlator_control.py

```diff
@@ -58,17 +62,31 @@
     parser.add_argument('--testing', help="Testing mode: do not use anything that "
                         "requires a connection to the correlator "
                         "(implies dryrun) [False]", action='store_true')
+    parser.add_argument('--lstlock', help="Lock observation start to a 16s LST grid"
+                        "[True]", action='store_true',default=True)
+    parser.add_argument('--now', help="Take data now."
+                        "autofills starttime, starttime_format and starttime_scale",
+                        action='store_true')
 
     args = parser.parse_args()
     db = mc.connect_to_mc_db(args)
     session = db.sessionmaker()
 
-    if args.starttime is not None:
-        starttime_obj = Time(args.starttime, format=args.starttime_format,
-                             scale=args.starttime_scale)
+    if args.command == 'take_data':
```

Comment:
wrapping the logic inside this conditional silently ignores inappropriate command line argument rather than telling the user that they are inappropriate. I favor more explicit error handling. The method this script wraps will provide useful error messages if you don't do this.

Reply:
removed "if take_data". I don't think its actually necessary.